frontend/packages/topology/src/components/graph-view/components/nodes/WorkloadNode.tsx

```diff
@@ -216,6 +219,8 @@ const WorkloadPodsNode: React.FC<WorkloadPodsNodeProps> = observer(function Work
           }
           attachments={nodeDecorators}
           contextMenuOpen={contextMenuOpen}
+          showAlertStatus={!!workloadRqAlert}
+          alertVariant={workloadRqAlertVariant}
```

Comment:
Please add the `workloadRqAlert` (`resourceQuotaAlert`?) to `getAggregateStatus` and add the warning before it fallbacks to `NodeStatus.default`, so it doesn't override the NodeStatus.danger.

```suggestion
```

Reply:
The reason I didnt use nodeStatus is because the other statuses like pod are visualised differently from how we want to show the alert statuses. 

If you see the `getAggregateStatus` status call is only made when `showDetails` is false  and now in the BaseNode once !showDetails is false `showStatusBackground` is set to true which applies the status modifier class to the node background which we do not want to do for the alert status.

When modifier is added to the node background
<img width="243" alt="Screenshot 2022-09-12 at 7 09 47 PM" src="https://user-images.githubusercontent.com/20724543/191763997-42ce561f-06aa-40ca-b958-1c9bcfcd502b.png">

When modifier is added on the boundry of the outer node circle
<img width="316" alt="Screenshot 2022-09-13 at 3 59 19 PM" src="https://user-images.githubusercontent.com/20724543/191764004-cd689615-e266-418c-9b66-2f13dbbd69a1.png">

Reply:
Thanks for the explanation, I though I have tested this but a new test confirms it doesn't looks as expected when changing just the node status, so I'm fine with a new prop. But I would like to ask you if we can apply just one? Will add a new comment for that.

Reply:
Done.